Webapp/sources/services/queries/workbook.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime, timedelta
+from typing import List
+
+import pytz
+from sources import models
+from sources.extensions import db
+
+
+def get_all_workbooks() -> List[models.WorkBook]:
+    """
+    Gets a list of workbooks created in the past 28 days
```

Comment:
This appears to just get all workbooks, so date time filter for the past 28 days?

Reply:
Good spot, I'll update the doc string. I thought for now at least, all workbooks would be more useful than just recently created ones